9.2/run-postgresql-slave.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -eu
+
+source ${HOME}/common.sh
+
+function initialize_replica() {
+  echo "Initializing replica"
+  chmod 0700 $PGDATA
+  local master=$(postgresql_master_addr)
+  PGPASSWORD="${POSTGRESQL_PASSWORD}" pg_basebackup -x --no-password --pgdata ${PGDATA} --host=${master} --port=5432 -U "${POSTGRESQL_USER}"
+
+  # PostgreSQL recovery configuration.
+  envsubst < ${POSTGRESQL_RECOVERY_FILE}.template > ${POSTGRESQL_RECOVERY_FILE}
+  cat >> "$PGDATA/recovery.conf" <<EOF
+
+# Custom OpenShift recovery configuration:
+include '${POSTGRESQL_RECOVERY_FILE}'
+EOF
+}
+
+check_env_vars
+generate_postgresql_config
+generate_passwd_file
+initialize_replica
+unset_env_vars
+
+# Testing
```

Comment:
I meant to test whether postgres can boot up correctly. Before that, there was something else that . Just realized it's not really needed now, will nuke it.